### ISSUE TITLE
Frozen During Vinewood Security System Hack

### ISSUE THREAD

**Issue by violetuniverse9** [echoroleplay/feedback#387]
**Description:** When disabling a Vinewood house's security system, the player will become stuck in place and unable to move IF the timer is allowed to run out. The bug does not occur if the hack is failed through errors. The timer must run out. In the three attempts I made, I was stuck every time and the alarm didn't sound. When I tried again and failed through errors, I was able to move again.

**Reproduction Steps:** Clearly describe the steps to reproduce the bug. 
1. Enter a Vinewood house
2. Attempt to disable the security system
3. Allow the timer to finish

**Expected Behavior:** I should be able to move and not be stuck in the phone animation.

**Actual Behavior:** I got stuck in the phone animation and no amount of canceling, pressing ESC, or using other animations freed me.

**Screenshot(s)/Video(s):** ![image](https://github.com/echoroleplay/feedback/assets/106116442/88060d51-6716-4342-9944-5a7e538ad94f)

**Comment by FlawwsX** [comment]
Do you have an F8 log?

**Comment by violetuniverse9** [comment]
> Do you have an F8 log?

No, but I can reproduce it and get you one. An F8 log is just a screenshot of the console right?

**Comment by violetuniverse9** [comment]
https://i.imgur.com/vXvzUNd.png
[CitizenFX_log_2023-07-19T003701.log](https://github.com/echoroleplay/feedback/files/12090069/CitizenFX_log_2023-07-19T003701.log)

**Comment by violetuniverse9** [comment]
It now doesn't let me reattempt the hack to unfreeze myself. It says that I'm already hacking. I have to relog to escape.

**Comment by DoowieEU** [comment]
Yeah, I've had this twice today.

From what I've noticed, and it seems to be the same for each time, the timer is below a set amount, which makes it impossible to complete, unless you have superhuman eye tracking and reflexes to get all 4 numbers completed.
